openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Users.spec.js

```diff
@@ -75,6 +96,10 @@ describe('Users flow should work properly', () => {
     softDeleteUser(userName);
     deleteSoftDeletedUser(userName);
   });
+
+  it('Search bot user', () => {
+    searchBotUser();
```

Comment:
Don't create function if it's not using from elsewhere
Also assertion shouldn't be in the function
Functions are just be helper and not testing anything